aea/protocols/generator.py

```diff
@@ -274,7 +358,7 @@ def _message_class_str(self) -> str:
                     covered.append(content_name)
                 cls_str += "    @property\n"
                 cls_str += "    def {}(self) -> {}:\n".format(
-                    content_name, content_type
+                    content_name, self._specification_type_to_python_type(content_type)
```

Comment:
Rather than doing the conversion again and again, why not do it once on the top and then reuse it?

Reply:
Done